client/src/components/applications/profil/components/MapContainer.js

```diff
@@ -1,89 +1,88 @@
 import React, {Component} from 'react'
 import {Map, Marker, GoogleApiWrapper} from 'google-maps-react'
-import './MapContainer.css'
 
 export class MapContainer extends Component {
-    mapStyles = [
-      {elementType: 'geometry.fill', stylers: [{color: '#f1f4f5'}]},
-      {elementType: 'geometry.stroke', stylers: [{color: '#cddae3'}]},
-      {elementType: 'labels.text.fill', stylers: [{color: '#666666'}]},
-      {elementType: 'labels.text.stroke', stylers: [{color: '#ffffff'}]},
-      {
-        featureType: 'administrative',
-        elementType: 'geometry.stroke',
-        stylers: [{color: '#333333'}],
-      },
-      {
-        featureType: 'landscape',
-        elementType: 'geometry.stroke',
-        stylers: [{color: '#859fb4'}],
-      },
-      {
-        featureType: 'landscape.natural',
-        elementType: 'geometry',
-        stylers: [{color: '#f1f4f5'}],
-      },
-      {
-        featureType: 'landscape.man_made',
-        elementType: 'geometry.fill',
-        stylers: [{color: '#dae3ea'}],
-      },
-      {
-        featureType: 'road',
-        elementType: 'geometry.fill',
-        stylers: [{color: '#ffffff'}],
-      },
-      {
-        featureType: 'road',
-        elementType: 'geometry.stroke',
-        stylers: [{color: '#a5baca'}],
-      },
-      {
-        featureType: 'road.local',
-        elementType: 'geometry.fill',
-        stylers: [{color: '#ffffff'}],
-      },
-      {
-        featureType: 'transit.line',
-        elementType: 'geometry',
-        stylers: [{color: '#c5d1da'}],
-      },
-      {
-        featureType: 'transit.station',
-        elementType: 'geometry',
-        stylers: [{color: '#e6ecf1'}],
-      },
-      {
-        featureType: 'water',
-        elementType: 'geometry.fill',
-        stylers: [{color: '#ebf8ff'}],
-      },
-      {
-        featureType: 'water',
-        elementType: 'labels.text.fill',
-        stylers: [{color: '#859fb4'}],
-      },
-      {
-        featureType: 'landscape.natural',
-        elementType: 'labels',
-        stylers: [{visibility: 'off'}],
-      },
-      {
-        featureType: 'road.highway',
-        elementType: 'labels',
-        stylers: [{visibility: 'off'}],
-      },
-      {
-        featureType: 'road.arterial',
-        elementType: 'labels',
-        stylers: [{visibility: 'off'}],
-      },
-    ]
+  mapStyles = [
```

Comment:
again colors in here - but regarding map as a whole, I think you may use the one from @Siegrift (contact him about this - they may be the same ? similar enough ? if not the same then should be made configurable a reused) - so that we don't have two components doing the same thing in one app